### PR TITLE
Verse Block: Add support for custom colors

### PR DESCRIPTION
**Pull request by scruffian**
## Description
Adds support for custom colors to the Verse bloc.

## How has this been tested?
In TT1 Blocks

## Screenshots 
Screenshot 2020-12-15 at 12 06 09

## Types of changes
New feature

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by paaljoachim** [comment]
Hey Kerry @gwwar can you do a review of Ben's PR to add color to the Verse block?

**Comment by gwwar** [comment]
@scruffian do you mind rebasing so I can manually test? Code here looks good so far 👍

**Review by kjellr — APPROVED** [review]
I rebased this PR, and it's working nicely for me:

Screen Shot 2021-04-12 at 10 11 40 AM

The `/verse/editor.scss` file only contained the single color rule which was removed by this PR, so I've gone ahead and deleted the file entirely. This could probably use one more set of 👀 before merge just to be sure that makes sense.

File: packages/block-library/src/verse/editor.scss [hunk 1]
```diff
@@ -1,3 +0,0 @@
-pre.wp-block-verse {
-	color: $gray-900;
```
Comment (gwwar) [inline packages/block-library/src/verse/editor.scss:2]: This is probably okay to remove. It didn't look like we were applying this to the published view. Not sure why we needed this one.

**Review by gwwar — APPROVED** [review]
Thanks for reviving this one @kjellr this one tests well for me too

**Comment by kjellr** [comment]
Thanks, everyone!

**Comment by scruffian** [comment]
Thanks for taking care of this - I've been AFK!

**Pull request merged**